packages/test-helpers/lib/web3.js

```diff
@@ -0,0 +1,130 @@
+module.exports = web3 => {
+
+  const GETH_NODE_ID = 'Geth'
+  const GANACHE_NODE_ID = 'TestRPC'
+
+  const isGeth = () => web3.version.node.includes(GETH_NODE_ID)
+
+  const isGanache = () => web3.version.node.includes(GANACHE_NODE_ID)
+
+  const advanceBlock = async () => {
+    await web3.currentProvider.send({
+      jsonrpc: '2.0',
+      method: 'evm_mine',
+      id: new Date().getTime()
+    })
+    return web3.eth.getBlock('latest').hash
+  }
+
+  const advanceBlocks = async blocks => {
+    for (let i = 0; i < blocks; i++) {
+      await advanceBlock()
+    }
+  }
+
+  const advanceTime = time => {
+    return web3.currentProvider.sendAsync({
+      jsonrpc: '2.0',
+      method: 'evm_increaseTime',
+      params: [time],
+      id: new Date().getTime()
+    })
+   }
+
+   const advanceTimeAndBlock = async time => {
+    await advanceTime(time)
+    await advanceBlock()
+    return web3.eth.getBlock('latest')
+  }
+  
+  const advanceTimeAndBlocksBy = async (time, numOfBlocks) => {
+    await advanceTime(time)
+    while (numOfBlocks !== 0) {
+      await advanceBlock()
+      numOfBlocks--
+    }
+    return web3.eth.getBlock('latest')
+  }
+
+  function signatures(contract, exclude, web3, names = false, excludeConstant = false) {
```

Comment:
I'm not super sure this util is relevant as a general util, and this type of thing is usually already done by the contract library that's being used (via an encoding function or otherwise).

What do you think about removing it?